datamodel/scripts/setup.sh

```diff
@@ -32,5 +32,5 @@ psql "service=${PGSERVICE}" -v ON_ERROR_STOP=1 -f ${DIR}/changelogs/0001/52_plan
 psql "service=${PGSERVICE}" -v ON_ERROR_STOP=1 -f ${DIR}/12_0_roles.sql
 psql "service=${PGSERVICE}" -v ON_ERROR_STOP=1 -f ${DIR}/12_1_roles.sql
 
-
+chmod +x ${DIR}/app/create_tww_app.py
```

Comment:
no, this needs to be done on your system (make the file executable)

Reply:
Which is a bit problematic when you work on windows and don't have the necessary rights to install git on your shell

Reply:
OK I managed that part. However, setup.sh does not treat create_tww_app as a python script anymore:

2024-02-09T06:54:04.6558778Z /src/datamodel/scripts/../app/create_tww_app.py: line 1: import: command not found
2024-02-09T06:54:04.6559590Z /src/datamodel/scripts/../app/create_tww_app.py: line 2: import: command not found
2024-02-09T06:54:04.6560370Z /src/datamodel/scripts/../app/create_tww_app.py: line 4: import: command not found
2024-02-09T06:54:04.6561163Z /src/datamodel/scripts/../app/create_tww_app.py: line 5: from: command not found
2024-02-09T06:54:04.6562024Z /src/datamodel/scripts/../app/create_tww_app.py: line 8: syntax error near unexpected token `('
2024-02-09T06:54:04.6563144Z /src/datamodel/scripts/../app/create_tww_app.py: line 8: `def run_sql_file(file_path: str, pg_service: str, variables: dict = {}):'

Reply:
you need to add `#!/usr/bin/env python` on top

Reply:
as in https://github.com/teksi/wastewater/pull/149/commits/875eb1d088e82f38d0378a2c51643b7ffb1792f4? Or do we need python3?